src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -572,6 +572,27 @@ describe('findBreakingChanges', () => {
     ]);
   });
 
+  it('should detect interfaces removed from types', () => {
+    const oldSchema = buildSchema(`
+      interface Interface1
+
+      interface Interface2 implements Interface1
+    `);
+
+    const newSchema = buildSchema(`
+      interface Interface1
+
+      interface Interface2
+    `);
+
+    expect(findBreakingChanges(oldSchema, newSchema)).to.deep.equal([
+      {
+        type: BreakingChangeType.INTERFACE_REMOVED_FROM_OBJECT,
```

Comment:
It's a problem that enum value is called `INTERFACE_REMOVED_FROM_OBJECT` because it was removed from interface. Since we targeting `15.0.0` we can do breaking change so I propose to rename it to `IMPLEMENTED_INTERFACE_REMOVED` similar to `FIELD_REMOVED`:
https://github.com/graphql/graphql-js/blob/21de98ea4855df8c5146580611fda8ad76f52a2f/src/utilities/findBreakingChanges.js#L45

Reply:
Great point! I've renamed this to `IMPLEMENTED_INTERFACE_REMOVED`.